pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -124,6 +125,8 @@ type OperationExecutor interface {
 func NewOperationExecutor(
 	operationGenerator OperationGenerator) OperationExecutor {
 
+	util.RegisterMetrics()
```

Comment:
hmm okay, I don't see a big downside either way but will change it to init lest I be accused of cargo culting. it's clear (for now), that operation_executor and pv controller are sole users of these metrics